docs/aws-iam-policies/README.md

```diff
@@ -1,17 +1,24 @@
 
-The directory contains files that correspond to IAM policies in the
-prod and community Fedora AWS accounts.
+This directory contains files that correspond to IAM policies in the
+community Fedora AWS account. The policies for the prod Fedora AWS
+account are stored in the [infra ansible repo](https://pagure.io/fedora-infra/ansible).
 
 #### Prod Account:
 
 - `fcos-builds-bot` user has the following policies attached
-    - [fcos-upload-amis](prod-account/fcos-upload-amis)
-    - [fcos-poc-artifacts](prod-account/fcos-poc-artifacts)
+    - [fcos-upload-amis.json](https://pagure.io/fedora-infra/ansible/blob/main/f/files/aws/iam/policies/fcos-upload-amis.json)
+        - Allows uploading/creating AMIs
+    - [fcos-poc-artifacts.json](https://pagure.io/fedora-infra/ansible/blob/main/f/files/aws/iam/policies/fcos-poc-artifacts.json)
```

Comment:
Aside: is the `poc` here for "proof of concept"? Obviously not high-priority, but we should drop that at some point.

Reply:
yep. I had the same exact thought, but I decided to not do the "dusty thing" for once and leave it (would need to PR upstream).